lite2/client.go

```diff
@@ -728,16 +733,21 @@ func (c *Client) sequence(
 		}
 
 		// Update trusted header and vals.
+		trustedHeader = interimHeader
```

Comment:
will this modify c.latestTrustedHeader each time?

Reply:
nope. Only at the end is c.lastTrustedHeader modified. Perhaps the naming should be changed if you think it is confusing. I just did it to save the need of declaring another variable

Reply:
> I just did it to save the need of declaring another variable

not sure. it's a pointer. when you modify it's value, you modify the pointer

Reply:
Yeah it will update it (as well as `latestTrustedNextVals` ). But the headers should be trusted by then